pkg/controller/cloneset/sync/cloneset_update.go

```diff
@@ -198,6 +206,22 @@ func (c *realControl) refreshPodState(cs *appsv1alpha1.CloneSet, coreControl clo
 	return false, res.DelayDuration, nil
 }
 
+func (c *realControl) fixPodTemplateHashLabel(cs *appsv1alpha1.CloneSet, pod *v1.Pod) (bool, error) {
```

Comment:
please comment the history and reason of why we need "fix" template hash. It seems that previously we only has controllerrevisionhash in the pod